src/map_object.c

```diff
@@ -1305,7 +1305,7 @@ void sub_0206294C (MapObject * mapObj, u32 param1)
     mapObj->unk_18 = param1;
 }
 
-u32 sub_02062950 (const MapObject * mapObj)
+u32 MapObject_GetEventType (const MapObject * mapObj)
```

Comment:
Label the underlying fields in these accessors when you name the functions themselves.